### ISSUE TITLE
Add instructions for finding files stored on S3

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/sc-data-integration#10]
In an effort to keep the files that are being used for analysis here in sync across data lab members, we should have an automated way to sync files from a central storage location (AWS S3) to a local computer to be able to interact with them within scripts in this repo. 
This would include the original data files as well as any files that are produced after processing (i.e. the output from converting loom files to SCEs or running scpca-downstream-analyses). 

Since we will want to have a way to ensure that everyone is working with the same files, we probably want a way to automate syncing both before starting an analysis session and after an analysis session, if any files have been changed. I _think_ a good approach would be to have a function that takes as input the list of files, the target directory, and the source directory and creates a call to aws cli to sync files. This function could then be used at the beginning of a script, i.e. to grab any SCE objects that are needed for processing, and then again at the end of a script to sync the outputs back with S3. 

An alternative thought to syncing the output back with S3 would be to use github actions to run the updated analysis upon merging a new PR to the main branch and then syncing outputs, but I don't know how feasible that actually is. 

Either way, I think it would be good to have a function in this repo that makes syncing easier and we can incorporate it into scripts.
